src/mngt/ocf_mngt_cache.c

```diff
@@ -541,10 +539,15 @@ static void _ocf_mngt_load_init_instance(ocf_pipeline_t pipeline,
 	if (ret)
 		OCF_PL_FINISH_RET(pipeline, ret);
 
-	if (context->metadata.shutdown_status == ocf_metadata_clean_shutdown)
-		_ocf_mngt_load_init_instance_clean_load(context);
-	else
-		_ocf_mngt_load_init_instance_recovery(context);
+
+	if (!context->cache->metadata.loaded) {
+		if (context->metadata.shutdown_status == ocf_metadata_clean_shutdown)
+			_ocf_mngt_load_init_instance_clean_load(context);
+		else
+			_ocf_mngt_load_init_instance_recovery(context);
+	} else {
+		_ocf_mngt_load_init_instance_complete(context, 0);
```

Comment:
In _ocf_mngt_load_init_instance_complete cleaning policy must be initialized with "init_meta" parameter set to false in case of context->cache->metadata.loaded